src/components/schedule-algorithm/schedule-algorithm.js

```diff
@@ -0,0 +1,66 @@
+const store = require('store');
+
+class ScheduleAlgorithm {
+
+  findInterval = (currentDate, timestamp) => {
```

Comment:
it's better to rename currentDate parameter here cause the same name is used below